CHANGELOG.md

```diff
@@ -2,5 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# [0.2.0] - 2018-03-15
```

Comment:
I'm almost tempted to bump this to `1.0.0` instead. This script has been super stable for a while now and this is a nice improvement. But either way wfm  👍

Reply:
Sure, why not!